ddd-koans/Entity/Identifier/task.html

```diff
@@ -0,0 +1,59 @@
+<html>
+<h2>Identifier</h2>
+<p>
+    In the previous parts of the course, the identification topic was briefly mentioned. The value object that is
+    identified by its value as described, however, the opposite situation was not presented.
+</p>
+<p>
+    The domain concept not identified by its values is called <i>Entity</i> in DDD. Any entity represents more than just
+    simple data container, it has its identity and must be identifiable somehow. Here comes the concept of the
+    identifier.
+</p>
+<p>
+    How can you identify a person? You might have guessed that with the given name and surname, but are you sure about
+    that?
+    People sometimes share given names and surnames. So governments around the world bestow multi-character codes upon
+    every
+    citizen to make the citizens distinguishable. It is sometimes your id card id number or insurance number.
+</p>
+<p>
+    The described above identifier is of artificial identifier type. The numbers are not directly connected with the
+    entities,
+    however, now they are distinguishable. They are typically generated by your software. Popular examples are UUIDs or
+    sequential ids get from a database.
+</p>
+<p>
+    Yet, natural identifiers also exist. They are typically provided by an external provider, for example, a user.
+    <!--    TODO(“Give some examples”)-->
```

Comment:
Each page in the book has its unique number? each day is identified by year, month and day in a month - a combination of all those attributes makes each day unique and identifiable?